CHANGELOG.md

```diff
@@ -7,11 +7,15 @@
 
 * Add additional constructor to `DataView` for `SharedArrayBuffer`.
   [#3695](https://github.com/rustwasm/wasm-bindgen/pull/3695)
+* Remove experimental status from `wasm_bindgen::module()` and update docs to say it works with `--target web`.
```

Comment:
```suggestion

* Stabilize `wasm_bindgen::module()`.
```
I don't think we ever added entries for documentation changes, so let's not start it here.